src/components/shop/shop-layout.js

```diff
@@ -79,4 +79,22 @@ export const getInnerRectBounds = scene => {
 };
 
 export const createRect = (scene, bounds, colour) =>
-    scene.add.rectangle(bounds.x, bounds.y, bounds.width, bounds.height, colour, 0.3);
+    scene.add.rectangle(bounds.x, bounds.y, bounds.width, bounds.height, colour, 0);
+
+export const createPaneBackground = (scene, bounds, pane) => {
+    const key = getPaneBackgroundKey(scene, pane);
+    const image = scene.add.image(0, 0, key);
+    image.setScale(bounds.width / image.width, bounds.height / image.height);
+    return image;
+};
+
+export const getPaneBackgroundKey = (scene, pane) => {
+    const {
+        assetPrefix,
+        assetKeys: { background },
+    } = scene.config;
+
+    if (typeof background === "string")
+        return background ? `${assetPrefix}.${background}` : `${assetPrefix}.noBackground`;
+    return background[pane] ? `${assetPrefix}.${background[pane]}` : `${assetPrefix}.noBackground`;
```

Comment:
I'm not really a fan of adding an empty image, to deal with the no background scenario (if that's what it is actually doing). Can we not just check whether a background key is provided before creating a background?

Reply:
Having a play with this. Yes it's definitely not the nicest way to do it. IIRC from Fri something broke if there was no image at all.

Reply:
Uhhhh yeah that one is weird. Basically at one point the menu was an object with a container and a couple of functions hanging off it. The fns use the container bounds to do layout on resize. But at this point everything that was in the container has been refactored out, apart from the background. If there's no background, a bunch of stuff breaks in the menu because it's an empty container and therefore has zero bounds. Could be changed but would need a large-ish refactor and I'm not sure it's worth it...

Reply:
Took a middle path on this: we still have a rect in there giving us an object with non-zero bounds to play with, but it no longer relies on a 16x16 transparent png file, so it's slightly less stinky. 3934321